app/routers/event.py

```diff
@@ -321,11 +320,14 @@ def get_participants_emails_by_event(db: Session, event_id: int) -> List[str]:
 
 def _delete_event(db: Session, event: Event):
     try:
+        # TODO: Check if user activate the restore deleted events feature
+
         # Delete event
-        db.delete(event)
+        # db.delete(event)
```

Comment:
So you don't delete it and instead leaves it without a connection?
Seems not good to me as if a lot of people will leave their events it will be too big for the db.

Reply:
I will delete the events after the FLAGS ticket will be merged,
now there is 30 days to delete permanently the events
when I use the flag, the user can chose if to activate this feature.
If the user deactivate this feature than the events will be deleted immediately